plugins/backstage-plugin-coder/src/components/CoderWorkspacesCard/WorkspacesListItem.tsx

```diff
@@ -226,6 +230,37 @@ export const WorkspacesListItem = ({
   );
 };
 
+const deletingStatuses: readonly WorkspaceStatus[] = ['deleting', 'deleted'];
+const offlineStatuses: readonly WorkspaceStatus[] = [
+  'stopped',
+  'stopping',
+  'pending',
```

Comment:
`starting` I think would also be technically offline?  At least, it is definitely not online.  Maybe we need another `pending` state separate from offline.

And maybe `canceling` and `canceled`.

Reply:
I'm a little less worried about `starting` since we're going to be sending the user to the Coder deployment UI anyway, but I can update the code to account for these